src/main/java/me/ryanhamshire/GPFlags/CommandHandler.java

```diff
@@ -498,6 +499,40 @@ public boolean onCommand(CommandSender sender, Command cmd, String commandLabel,
             GPFlags.sendMessage(player, color, result.message.messageID, result.message.messageParams);
             if (result.success) plugin.getFlagManager().save();
 
+            if (args[0].equalsIgnoreCase("NoEnter") && args.length >= 2) {
+                World world = player.getWorld();
+                for (Player p: world.getPlayers()) {
+                    if (claim.contains(p.getLocation(), true, false)) {
+                        if (claim.getPermission(p.getName()) == null && !claim.getOwnerName().equals(p.getName())) {
+                            GriefPrevention.instance.ejectPlayer(p);
+                        }
+                    }
+                }
+            }
+            if (args[0].equalsIgnoreCase("NoEnterPlayer") && args.length >= 2) {
+                for (int i = 1; i < args.length; i++) {
+                    Player target = Bukkit.getPlayer(args[i]);
+                    if (target != null && target.getName().equals(args[i])) {
+                        if (claim.contains(target.getLocation(), true, false)) {
+                            if (claim.getPermission(args[i]) == null) {
+                                GriefPrevention.instance.ejectPlayer(target);
+                            }
+                        }
+                    }
+                }
+            }
+            if (args[0].equalsIgnoreCase("OwnerFly")) {
+                player.setAllowFlight(true);
+            }
+            if (args[0].equalsIgnoreCase("OwnerMemberFly")) {
+                World world = player.getWorld();
+                for (Player p: world.getPlayers()) {
+                    if (claim.contains(p.getLocation(), true, false)) {
+                        p.setAllowFlight(true);
+                    }
```

Comment:
Correct me if I'm wrong, but isn't this flag for members and not all players?
What if someone is just visiting your claim, they would end up getting fly also.
Shouldn't this account for that?